embark

```diff
@@ -0,0 +1,37 @@
+set notification-error-bg           "#1E1C31"
+set notification-error-fg           "#F48FB1"
+set notification-warning-bg         "#1E1c31"
+set notification-warning-fg         "#FFE6B3"
+set notification-bg                 "#1E1C31"
+set notification-fg                 "#CBE3E7"
+
+set completion-bg                   "#1E1C31"
+set completion-fg                   "#CBE3E7"
+set completion-group-bg             "#87DFEB"
```

Comment:
Can we change this to #A1EFD3

Reply:
Sure that would look close to the original